sql/2020/06_THIRD_PARTIES/percent_of_pages_with_third_party.sql

```diff
@@ -0,0 +1,55 @@
+#standardSQL
+# Percent of pages that include at least one third party resource.
+
+WITH
+  requests AS (
+  SELECT
+    'desktop' AS client,
+    page,
+    url
+  FROM
+    `httparchive.almanac.requests_desktop_1k`
+  UNION ALL
+  SELECT
+    'mobile' AS client,
+    page,
+    url
+  FROM
+    `httparchive.almanac.requests_mobile_1k` ),
+
+  thirdParty AS (
+  SELECT
+    domain
+  FROM
+    `lighthouse-infrastructure.third_party_web.2020_05_01`)
```

Comment:
@rviscomi didn't you create a copy of the september data in httparchive namespace somewhere?

Reply:
Yes, `httparchive.almanac.third_parties`